docs/OnlyOffice.rst

```diff
@@ -0,0 +1,137 @@
+.. _OnlyOffice:
+
+#############################
+OnlyOffice Connector for eXo
+#############################
+
+**What is OnlyOffice**
+
+`OnlyOffice Online Editors <https://www.onlyoffice.com/office-suite.aspx>`__ is a great open source solution 
+that lets you to create, edit  and collaborate online on your office documents. 
+OnlyOffice editors are 100% compatible with Microsoft Office document formats : docx, pptx and xlsx.
+
+|image0|
+
+
+.. note:: eXo partnered with OnlyOffice to offer the enterprise-ready version of OnlyOffice editors to its customers along 
+          with the **commercial editions of eXo Platform**. Thus, eXo customers are required to purchase a dedicated 
+          subscription from eXo to run the OnlyOffice connector add-on on their eXo Platform instance. 
+          However, a 30-day trial is prepackaged with the connector in order to test-drive the integration before purchase.
+
+To install the add-on, use the following command:
+
+::
+
+		./addon install exo-onlyoffice
+		
+
+.. _HowToUse:
+
+================================
+How to use OnlyOffice for eXo?
+================================	
+
+.. _EditOnline_BTN:
+
+Edit Online button
+~~~~~~~~~~~~~~~~~~~~
+		
+Having satisfied the above conditons and installed the add-on, you can easily edit your Microsoft Office documents online.
+Just hit the ``Edit Online`` button |image1|.
+
+The ``Edit Online`` button is available in these locations:
+
+-  In the activity stream among the possible actions available on the document
+
+   |image2|
+   
+-  In the document preview interface
+
+   |image3|
+
+-  In the ``Documents`` application, in the action bar
+
+   |image4|
+   
+Clicking on this button loads the dcument in the corresponding OnlyOffice editor in new tab for maximum comfort.
+
+
+.. _EditorInterface:
+
+Editor Interface
+~~~~~~~~~~~~~~~~~~
+
+Clicking on |image5| button opens the document you want to edit in an another tab.
+The interface contains the content of the document loaded in the OnlyOffice editor with a floating button.
+
+Clicking on that button opens a drawer "Save version" with these fields:
```

Comment:
Is "drawer" a understandable term for everyone ?

Reply:
I think yes! any other proposals ?

Reply:
If you think it is understandable, that's fine for me :) Otherwise, we could use "panel" maybe.